tests/sqlparser_common.rs

```diff
@@ -6795,7 +6795,8 @@ fn parse_create_view_with_options() {
 #[test]
 fn parse_create_view_with_columns() {
     let sql = "CREATE VIEW v (has, cols) AS SELECT 1, 2";
-    match verified_stmt(sql) {
+    // TODO: why does this fail for ClickHouseDialect?
```

Comment:
Could you perhaps file a ticket to track this isse?

Reply:
Will do!